backend/strapi-sync/create-multiple-item-campgrounds.js

```diff
@@ -0,0 +1,438 @@
+import { Campground, Feature, Park } from "../models/index.js";
+
+import { getItemByAttributes, createModel } from "./utils.js";
+
+const campgrounds = [
```

Comment:
Is this hardcoded list customized? Or can it come from Strapi somehow?

Reply:
It has to be hard-coded because it was created based on "made up" arbitrary rules. The data was manually reviewed to create this list. It helps to keep track the strapiId of the features we are changing. 
Also, I think Brett made some edits, so this will need to be updated.

Reply:
Got it 👍 And we had discussed before, if campsite groupings change in the future, hopefully we can update our db first and be the source of truth, rather than changing Strapi first and throwing things out of whack in our db.